docs/content/documentation/systems/Materials/tensor_mechanics/ComputeInstantaneousThermalExpansionFunctionEigenstrain.md

```diff
@@ -1,15 +1,21 @@
-#ComputeInstantaneousThermalExpansionFunctionEigenstrain
+#Compute Instantaneous Thermal Expansion Function Eigenstrain
 
 !syntax description /Materials/ComputeInstantaneousThermalExpansionFunctionEigenstrain
 ## Description
 
-This model computes the eigenstrain tensor resulting from isotropic thermal expansion where the temperature-dependent thermal expansion is defined by a user-supplied function that describes the instantaneous thermal expansion coefficient $\alpha$ as a function of temperature, $T$.  Using a trapezoidal rule to perform time integration of this function, the current value of the thermal eigenstrain tensor, $\boldsymbol{\epsilon}^{th}_{t+1}$ is computed at a given time as:
+This model computes the eigenstrain tensor resulting from isotropic thermal expansion where the temperature-dependent thermal expansion is defined by a user-supplied function that describes the instantaneous thermal expansion coefficient $\alpha$ as a function of temperature, $T$.  Using a trapezoidal rule to perform time integration of this function, the current value of the thermal eigenstrain tensor, $\mathbf{\epsilon}^{th}_{t+1}$ is computed at a given time as:
```

Comment:
\mathbf doesn't actually do anything to Greek characters. The fact that KaTeX doesn't yet support \boldsymbol is a big problem. We need some alternative way of denoting tensors with Greek characters.

Reply:
Looks like KaTeX should have this support shortly: https://github.com/Khan/KaTeX/pull/1011 and https://github.com/Khan/KaTeX/issues/633...I'm not sure what the wait time is until we can use `\boldsymbol` here